open-metadata-implementation/common-services/generic-handlers/src/main/java/org/odpi/openmetadata/commonservices/generichandlers/SchemaAttributeHandler.java

```diff
@@ -1865,6 +1874,166 @@ public void   updateSchemaAttribute(String               userId,
+                        {
+                            for (Classification classification : classifications)
+                            {
+                                if (classification.getName().equals(OpenMetadataAPIMapper.CALCULATED_VALUE_CLASSIFICATION_TYPE_NAME)) {
+                                    removeClassificationFromRepository(userId,
+                                                                       externalSourceGUID,
+                                                                       externalSourceName,
+                                                                       schemaAttributeGUID,
+                                                                       schemaAttributeGUIDParameterName,
+                                                                       OpenMetadataAPIMapper.SCHEMA_ATTRIBUTE_TYPE_NAME,
+                                                                       OpenMetadataAPIMapper.CALCULATED_VALUE_CLASSIFICATION_TYPE_GUID,
+                                                                       OpenMetadataAPIMapper.CALCULATED_VALUE_CLASSIFICATION_TYPE_NAME,
+                                                                       forLineage,
+                                                                       forDuplicateProcessing,
+                                                                       effectiveTime,
+                                                                       methodName);
+                                }
+                            }
+                        }
+                    }
+                } else {
+                    setClassificationInRepository(userId,
+                                                  externalSourceGUID,
+                                                  externalSourceName,
+                                                  schemaAttributeEntity,
+                                                  schemaAttributeGUIDParameterName,
+                                                  attributeTypeName,
+                                                  OpenMetadataAPIMapper.CALCULATED_VALUE_CLASSIFICATION_TYPE_GUID,
+                                                  OpenMetadataAPIMapper.CALCULATED_VALUE_CLASSIFICATION_TYPE_NAME,
+                                                  schemaAttributeBuilder.getCalculatedValueProperties(formula, methodName),
+                                                  isMergeUpdate,
+                                                  forLineage,
+                                                  forDuplicateProcessing,
+                                                  supportedZones,
+                                                  effectiveTime,
+                                                  methodName);
+
+                }
+                // todo this logic assumes the schema type is stored as a classification
+                setClassificationInRepository(userId,
```

Comment:
Line up parameters under userId

Reply:
I do not see that this is resolved - the parameters are too clase to the left margin